tests/cpp/test_matmul_scheduler.cpp

```diff
@@ -2685,6 +2685,50 @@ TEST_F(NVFuserTest, SegmentLinearOpPrologue) {
   testValidate(executor_cache.fusion(), outputs, {t0, t1}, __LINE__, __FILE__);
 }
 
+// Test that the matmul scheduler refuses to translate a matmul that is not
+// Half or BFloat16
+TEST_F(NVFuserTest, SegmentMatmulOpUnsupportedDtype) {
```

Comment:
nit: `NVFuserTest`-> `MatmulSchedulerTest`. Same for the two tests above. It helps with filtering.